src/stream.c

```diff
@@ -258,6 +258,11 @@ static void rtp_handler(const struct sa *src, const struct rtp_header *hdr,
 			     sdp_media_name(s->sdp), s->ssrc_rx, hdr->ssrc,
 			     mbuf_get_left(mb), src);
 		}
+		else if (hdr->m) {
+			flush = true;
+			info("stream: %s: marker bit set\n",
+				sdp_media_name(s->sdp));
+		}
```

Comment:
the marker bit has different semantics for audio/video stream.

audio: signal start of a new talk spurt
video: signal a complete video frame

I think checking RTP header marker bit should be done in audio.c instead.

Reply:
Right! Now it looks better.